toolkit/scripts/telemetry_hopper/telemetry_hopper.py

```diff
@@ -159,12 +160,23 @@ def _create_spans(self, request) -> List[SpanData]:
 def extract_attribute_value(value_proto: Any) -> Optional[Any]:
     """Extract value from protobuf AnyValue."""
     value_case = value_proto.WhichOneof("value")
+
     value_mapping = {
         "string_value": value_proto.string_value,
         "int_value": value_proto.int_value,
         "double_value": value_proto.double_value,
         "bool_value": value_proto.bool_value,
     }
+
+    if value_case == "array_value":
+        # Handle array values by extracting each item using the same mapping
+        array_values = []
+        for item in value_proto.array_value.values:
+            item_case = item.WhichOneof("value")
```

Comment:
Would it make sense to recursively call `extract_attribute_value` here?

Reply:
good point, updated